flake.nix

```diff
@@ -106,7 +106,12 @@
                 pkgs.darwin.apple_sdk.frameworks.Security
                 pkgs.darwin.apple_sdk.frameworks.SystemConfiguration
                 pkgs.darwin.apple_sdk.frameworks.CoreFoundation
-              ];
+              ] ++ (with inputs'.holonix.packages; [
+                bootstrap-srv
+                lair-keystore
+                hc
+                hc-launch
```

Comment:
We shouldn't need `hc-launch`, and the bootstrap-srv and lair-keystore should already be present above. `hc` does need to be added but it should be added above with the other Holonix inputs I think